app/models/slack_presence_check.rb

```diff
@@ -4,4 +4,10 @@ class SlackPresenceCheck < ApplicationRecord
   enum :presence, [:active, :away]
 
   validates_presence_of :check_time
+
+  paginates_per 50
+
+  def self.collect_for_pagination
+    self.all.includes(:student).order(id: :desc)
```

Comment:
Let's change this to order by recently created_at

Reply:
Done in latest commits!